app/src/main/java/com/example/quiz/MainActivity.java

```diff
@@ -136,6 +131,28 @@ public void onClick(View view) {
 
     }
 
+    private void initViews() {
```

Comment:
Entire view initializing findViewById calls moved to a common method for easy access.